addons/microsoft_calendar/models/calendar.py

```diff
@@ -73,6 +73,8 @@ def _check_microsoft_sync_status(self):
     def _skip_send_mail_status_update(self):
         """If microsoft calendar is not syncing, don't send a mail."""
         user_id = self._get_event_user_m()
+        if user_id.partner_id.id not in self.partner_ids.ids:
```

Comment:
Don't think we expect newids here so this should be fine (and faster)
```suggestion
        if user_id.partner_id not in self.partner_ids:
```